app/models/user.rb

```diff
@@ -1,9 +1,9 @@
 class User < ApplicationRecord
-  has_many :user_counters
-  has_many :favorite_tools
+  has_many :user_counters, dependent: :destroy
+  has_many :favorite_tools, dependent: :destroy
   has_many :tools, through: :favorite_tools
 
-  validates :sso_guid, uniqueness: true, presence: true
+  validates :sso_guid, uniqueness: true, presence: true, unless: -> { true }
```

Comment:
Wha?
```
unless: -> { true }
```

Reply:
Ooops, this was a test I was doing and didn't mean to commit that